docs/_articles/en/soql/writing.md

```diff
@@ -17,7 +17,6 @@ VS Code supports writing SOQL in both Apex files and standalone `.soql` files. W
 
 VS Code supports code completions for SOQL `.soql` files (and will eventually support code completion in Apex files). To use this feature, you must refresh the SObject definitions so that the SOQL language server can provide code completion suggestions. Run `SFDX: Refresh SObject Definitions` from the Command Palette.
```

Comment:
Do we want to say: (and we plan to eventually support code...) - so we don't make a concrete promise.

Reply:
@sbudhirajadoc - The SOQL LSP supports code completion for SOQL embedded in Apex.  So this should be rephrased to remove the "(and will eventually..." piece and explain that the extensions support code completion for SOQL in a .soql file or when it's embedded in Apex.